### PR TITLE
Update README.md with Exposed Port check as part of "getting started"

### PR DESCRIPTION
**Pull request by slernersplunk**
Added link about exposed ports to "Getting started"... because its part of getting started...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,6 +88,7 @@ All you need to get started is:
 - [Splunk Access Token](https://docs.splunk.com/Observability/admin/authentication-tokens/org-tokens.html#admin-org-tokens)
 - [Splunk Realm](https://dev.splunk.com/observability/docs/realms_in_endpoints/)
 - [Agent or Gateway mode](docs/agent-vs-gateway.md)
+- [Doublecheck exposed ports](https://github.com/signalfx/splunk-otel-collector/blob/main/docs/security.md#exposed-endpoints) to make sure your environment doesn't have conflicts- these can be changed via Collector Agent's configuration
```
Comment (breedx-splk) [inline README.md:91]:
```suggestion
- [Double-check exposed ports](https://github.com/signalfx/splunk-otel-collector/blob/main/docs/security.md#exposed-endpoints) to make sure your environment doesn't have conflicts. Ports can be changed in the collector's configuration.
```

**Review by breedx-splk — CHANGES_REQUESTED** [review]
Small change recommended.  Thanks.

File: README.md (hunk 1 above)
Comment (slernersplunk) [reply README.md:91]: thumbsup

**Review by pjanotti — APPROVED** [review]
@slernersplunk adding the info and trying to prevent the issue is good, but, if any component is not giving a clear error message about port already in use please let us know.

**Pull request merged**